redaxo/src/addons/structure/lib/structure_context.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * @package redaxo5
```

Comment:
Diese Annotation schwirrt teils noch in den Klassen, verwenden wir aber eigentlich nicht mehr. Daher bitte entfernen.

Stattdessen direkt an die Klasse die `@package redaxo\structure`-Angabe.